docs/source/experiment/training_service.rst

```diff
@@ -0,0 +1,18 @@
+Training Services
+=================
+
+NNI has supported many training services listed below. Users can go through each page to learning how to configure the corresponding training service. NNI has high extensibility by design, users can customize new training service for their special resource, platform or needs.
+
+..  toctree::
```

Comment:
Suggest hiding the `toctree` with

```
.. toctree::
   :hidden:
```

And make another table like:

| Platform | Description (this header can be omitted) |
|------|-----|
| Local | Running trials on one single local machine. |
| Remote | .... |

Reply:
updated